public/data/Shivendu-Kumar.json

```
@@ -0,0 +1,23 @@
+{
+    "name": "Shivend Kumar",
+    "type": "personal",
+    "bio": "Open Source Enthusiast!",
+    "avatar": "https://github.com/Shivendu-Kumar.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/Shivendu-Kumar",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/shivendu_kr_jha",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/shivendu-kumar-731a46214/",
+        "icon": "linkedin"
+      }
+    ]
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```